Formula/quick-lint-js.rb

```diff
@@ -0,0 +1,29 @@
+class QuickLintJs < Formula
+  desc "Find bugs in your JavaScript code"
+  homepage "https://quick-lint-js.com/"
+  url "https://c.quick-lint-js.com/releases/2.2.0/source/quick-lint-js-2.2.0.tar.gz"
+  sha256 "13bda2ff0c286d3e32053b93de723fda8caaa1d33849233e9502d16bb858815e"
+  license "GPL-3.0-or-later"
+  head "https://github.com/quick-lint/quick-lint-js.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  fails_with :clang do
+    build 1100  # Xcode 11.3.1
+    cause "Boost.JSON doesn't like Clang's std::string_view"
```

Comment:
The quick-lint-js tarball includes its own copy of Boost.

Reply:
Since there seems to be a Github pages outage let's link it like: https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md#stuff-that-requires-vendored-versions-of-homebrew-formulae

Reply:
@SMillerDev I agree with Homebrew's policy. I'll work on making quick-lint-js' build system use Homebrew's versions.